zulip_bots/zulip_bots/bots/xkcd/test_xkcd.py

```diff
@@ -15,10 +15,10 @@ def test_bot(self, mock_logging_exception):
         help_txt = "xkcd bot supports these commands:"
         err_txt  = "xkcd bot only supports these commands:"
         commands = '''
-* `@xkcd help` to show this help message.
-* `@xkcd latest` to fetch the latest comic strip from xkcd.
-* `@xkcd random` to fetch a random comic strip from xkcd.
-* `@xkcd <comic id>` to fetch a comic strip based on `<comic id>` e.g `@xkcd 1234`.'''
+* `@**test bot** help` to show this help message.
+* `@**test bot** latest` to fetch the latest comic strip from xkcd.
+* `@**test bot** random` to fetch a random comic strip from xkcd.
+* `@**test bot** <comic id>` to fetch a comic strip based on `<comic id>` e.g `@**test bot** 1234`.'''
```

Comment:
Maybe .format('@**test bot**'), like in the xkcd bot itself? Or even dynamically fetch the bot's name in the test? But nowt a must.

Reply:
This could be factored-out, but I'm not sure for what purpose, unless we expect the test name to change?

Rather than just take a fixed name in the test lib, we could take a similar approach as in the default methods PR, with an optional class variable in each bot-test-class (ie. like `bot_name`) to specify the `BotIdentity`? It could default to something, but also specified there, so there would be consistency and the `.format` approach could reference the BotIdentity variable too, in the bot-specific test?